CHANGES.rst

```diff
@@ -3,13 +3,22 @@ Version 0.6.0
 
 New features
 ^^^^^^^^^^^^
+- HelioPy has been integrated with SunPy TimeSeries and AstroPy Units. All of
+  HelioPy modules now return units.
+- Added a new :func:`.data.util.cdf_units` function that can extract UNIT attribute from
+  the CDF files.
 
 Backwards incompatible changes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - :func:`.data.helios.merged`, :func:`.data.helios.mag_4hz`,
   :func:`.data.helios.corefit` and :func:`.data.helios.mag_ness` no longer take
   a `verbose` keyword argument. :issue:`467`
+- HelioPy no longer returns a Pandas DataFrame.
 
+Fixed bugs
+^^^^^^^^^^
+- :func:`.data.mms.fpi_dis_moms` has been updated to the V3.3.0.
```

Comment:
I put this fix in the recently released v0.5.2, so no need to put a chnagelog entry here.